docs/markdown/Reference-manual.md

```diff
@@ -1579,11 +1580,13 @@ arguments:
    that override those set in the subproject's `meson_options.txt`
    (like `default_options` in `project`, they only have effect when
    Meson is run for the first time, and command line arguments override
-   any default options in build files). *(since 0.54.0)*: `default_library`
-   built-in option can also be overridden.
- - `version`: works just like the same as in `dependency`.
-   It specifies what version the subproject should be, as an example `>=1.0.1`
- - `required` *(since 0.48.0)*: By default, `required` is `true` and
+   any default options in build files).
+   *Since 0.54.0* `default_library` built-in option can also be overridden.
+   *Since 0.56.0* a Dictionary may be used instead of an array
```

Comment:
here you have a 3rd way of adding dictionary documentation. Would be nice to harmonize across all default_options docs. FWIW, this is the format I like the more.